pylon/src/nginx.conf.template

```diff
@@ -74,12 +74,90 @@ http {
     }
 
     #
-    # PAI web portal
+    # Web portals
     #
 
-    location ~ ^(.*) {
+    # Kubernetes dashbaord
+    location ~ ^/kubernetes/dashboard$ {
+      return 307 $scheme://$http_host$request_uri/;
+    }
+    location ~ ^/kubernetes/dashboard(.*)$ {
```

Comment:
Why we need this one?

Reply:
Can you add a comment for the purpose of this 307 redirections?

---
In reply to: [170492970](https://github.com/Microsoft/pai/pull/336#discussion_r170492970) [](ancestors = 170492970)

Reply:
Sure. Done.